docs/topics/OAuth2.md

```diff
@@ -19,36 +19,23 @@ The first step in implementing OAuth2 is [registering a developer application](#
 
 ###### OAuth2 Scopes
 
-These are a list of all the OAuth2 scopes that Discord supports. Some scopes require approval from Discord to use. Requesting them from a user without approval from Discord may cause errors or undocumented behavior in the OAuth2 flow.
+These are a list of all the publicly available OAuth2 scopes that Discord supports.
 
 | Name                                     | Description                                                                                                                                                                             |
 | ---------------------------------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
-| activities.read                          | allows your app to fetch data from a user's "Now Playing/Recently Played" list - requires Discord approval                                                                              |
-| activities.write                         | allows your app to update a user's activity - requires Discord approval (NOT REQUIRED FOR [GAMESDK ACTIVITY MANAGER](#DOCS_GAME_SDK_ACTIVITIES/))                                       |
 | applications.builds.read                 | allows your app to read build data for a user's applications                                                                                                                            |
-| applications.builds.upload               | allows your app to upload/update builds for a user's applications - requires Discord approval                                                                                           |
 | applications.commands                    | allows your app to use [commands](#DOCS_INTERACTIONS_APPLICATION_COMMANDS/) in a guild                                                                                                  |
 | applications.commands.update             | allows your app to update its [commands](#DOCS_INTERACTIONS_APPLICATION_COMMANDS/) using a Bearer token - [client credentials grant](#DOCS_TOPICS_OAUTH2/client-credentials-grant) only |
 | applications.commands.permissions.update | allows your app to update [permissions for its commands](#DOCS_INTERACTIONS_APPLICATION_COMMANDS/permissions) in a guild a user has permissions to                                      |
-| applications.entitlements                | allows your app to read entitlements for a user's applications                                                                                                                          |
-| applications.store.update                | allows your app to read and update store data (SKUs, store listings, achievements, etc.) for a user's applications                                                                      |
```

Comment:
SKUs still exist

Reply:
That is not the point here – the point is you cannot reasonably expect to use the store endpoints that scope grants access to.